src/common/playlists/FeaturedPlays.jsx

```diff
@@ -1,21 +1,78 @@
+import React, { Fragment, useEffect, useState } from 'react';
 import PlayThumbnail from './PlayThumbnail';
-import useFeaturedPlays from 'common/hooks/useFeaturedPlays';
+import { ReactComponent as ImageOops } from 'images/img-oops.svg';
+import Loader from 'common/spinner/spinner';
+import './playlist.css';
+import { useLocation } from 'react-router-dom';
+import { fetchPlays } from './FeaturedPlaysFetcher/PlayFetcher';
 
 const FeaturedPlays = () => {
-  const [loading, error, data] = useFeaturedPlays();
-  const success = !loading && !error && !!data.length;
+  const [loading, setLoading] = useState(false);
```

Comment:
@Shivam-Katare We should modify the useFeaturedPlays hook as much as possible to get the featured plays... not putting the logic here.